docs/pages/off-canvas.md

```diff
@@ -346,13 +346,17 @@ For an example of off-canvas, checkout this top bar with off-canvas navigation a
 
 ## In-Canvas to Off-Canvas
 
-With this feature you can have a standard page element move off-canvas at a particular breakpoint. Use the new class <code>.in-canvas-for-[BREAKPOINT]</code> for this. This differs from the <a href="#reveal-on-larger-screens">Reveal on Larger Screens</a> feature it doesn't actually open the off-canvas for specific screen sizes but overrides the off-canvas styles so it behaves as a regular page element. This way you can place an element anywhere on the page and move it into off-canvas for e.g. small screens.
+With this feature you can have a standard page element move off-canvas at a particular breakpoint. Use the new class <code>.in-canvas-for-[BREAKPOINT]</code> and the <code>inCanvasOn</code> option for this. This differs from the <a href="#reveal-on-larger-screens">Reveal on Larger Screens</a> feature as it doesn't actually open the off-canvas for specific screen sizes but overrides the off-canvas styles so it behaves as a regular page element. This way you can place an element anywhere on the page and move it into off-canvas for e.g. small screens only.
```

Comment:
You can replace `Use _the new_ class...` btw. It's 10-months old and already in `v6.4.0`. Also, options are the standard and recommanded "way to go" to customize components, so I think you could put it first.

Something like:
> You can make a standard page element move off-canvas at a particular breakpoint with the `inCanvasOn` option (or the `.in-canvas-for-*` classname).

Reply:
Yup the 'new' is definitely not up-to-date anymore – I've adjusted the text. 
I've removed the usage of the `.in-canvas-for-[BREAKPOINT]` class from the text/example and mention it only in a callout anymore because usually only the  JS option should be used.